### ISSUE TITLE
Adaptive data_limit

### ISSUE THREAD

**Issue by chmanu** [darold/ora2pg#729]
Hello,
I suggest a enhanced behavior by adding an adaptive data limit parameter.
In my last migration, I had many big tables but some of them had the avg_row_len at 80bytes where some others had an avg_row_len at 4000bytes. In that mode, I had to calculate the data limit to take into account for each different type of table and launched different instances of ora2pg in order to avoid any swap processing.
I suggest a parameter called adaptive_data_limit (in bytes) which can replace the data_limit for each table based on the avg_row_len in the dba_tables.

Regards,
Manuel Pavy

**Comment by darold** [comment]
Good idea. There is already an adaptive data_limit when Ora2Pg detect that the table has a BLOB column it can be adapted to the value of avg_row_len.
